test/mocha.opts

```diff
@@ -1,4 +1,4 @@
 --compilers js:babel/register
---recursive
+--recursive test/tests
```

Comment:
It will, but it will read in all the other files as tests as well. Which isn't really harmful, but just seemed wasteful.